packages/react/src/Card/Card.test.tsx

```diff
@@ -130,4 +130,20 @@ describe('Card', () => {
     const cardEl = getByTestId(mockTestId).firstChild
     expect(cardEl).toHaveClass(classToCheck)
   })
+
+  it('renders the image correctly into the document', () => {
+    const mockTestId = 'test'
+    const classToCheck = 'Card__image'
+
+    const {getByTestId} = render(
+      <Card href={mockHref} data-testid={mockTestId}>
+        <Card.Image src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png" alt=" " />
+        <Card.Heading>{mockHeading}</Card.Heading>
+        <Card.Description>{mockDescription}</Card.Description>
+      </Card>
+    )
+
+    const cardEl = getByTestId(mockTestId).firstChild
```

Comment:
Nit. Could we use `getByAltText` to encourage using accessible matchers? Also prevents brittle tests, where firstChild is coincidental. Should be some examples in the Image component test file.